Gemfile

```diff
@@ -91,7 +91,7 @@ gem "http"
 
 # Database
 
-gem "pg", "~> 1.4.3"
+gem "pg", "~> 1.5.3"
```

Comment:
@tanmoysrt do we need this update?

Reply:
No, ig during installing the gem it got update. I will check and will revert the changes if not required